spec/human_player_spec.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'human_player'
+require 'user_input'
+require 'fake_io'
+require 'board'
+require 'presenter'
+
+describe HumanPlayer do
+  let(:human) do
+    HumanPlayer.new(token: 'X',
+                    user_input: UserInput.new(io: FakeIO.new(input_stream: '1')))
+  end
+
+  describe '#token' do
+    it 'returns a string' do
+      expect(human.token).to be_kind_of String
```

Comment:
I'm not sure if we need this test. `human.token` isn't a method call that has logic we want to test.